specification/dartLangSpec.tex

```diff
@@ -2963,17 +2963,36 @@ \subsection{Constructors}
 the instance scope of $C$, where $X_1, \ldots, X_m$ are in scope.
 }
 
-\commentary{
+\commentary{%
 A constructor declaration may conflict with static member declarations
-(\ref{classMemberConflicts}).
+(\ref{classMemberConflicts}).%
 }
 
 % In what scope do constructors go? The simple names of named constructors go in the static scope of the class. Unnamed ones go nowhere, but we use the class name to refer to them; the class name could also in the static scope of the class as well to prevent weird errors, or we could ban it explicitly and avoiding duplication. Similarly, the instance scope could contain the constructor names and class name, or we could have special rules to prevent collisions between instance members and constructors or the class.
 
 % The enclosing scope of a generative constructor is the instance scope of the class in which it is declared (but what about redirecting?)
 
 \LMHash{}%
-If{}f no constructor is specified for a class $C$, it implicitly has a default constructor \code{C() : \SUPER{}() \{\}}, unless $C$ is class \code{Object}.
+If{}f no constructor is specified for a class $C$,
+it implicitly has a default constructor \code{C():\ \SUPER();}
+unless $C$ is the built-in class \code{Object}.
+
+\LMHash{}%
+If a class $C$ has no generative constructors
```

Comment:
And then here clarify, "no generative constructors (either explicitly declared or the implicit default one above)"

Reply:
Done.